python/katana/local/import_data.py

```diff
@@ -21,6 +28,56 @@
 ]
 
 
+class EntityTypeArray(EntityTypeManager):
```

Comment:
Love this class - thanks for doing this!

Reply:
Yea. I needed it, but I was pretty sure it would be generally useful.

Oh. I should add it to `__all__`.